### PR TITLE
Added BorMainnet

### PR DESCRIPTION
**Pull request by Uttam-Singhh**

### CODE REVIEW

File: cmd/geth/consolecmd.go [hunk 1]
```diff
@@ -137,6 +137,9 @@ func remoteConsole(ctx *cli.Context) error {
 			} else if ctx.GlobalBool(utils.MumbaiFlag.Name) {
 				homeDir, _ := os.UserHomeDir()
 				path = filepath.Join(homeDir, "/.bor/data")
+			} else if ctx.GlobalBool(utils.BorMainnetFlag.Name) {
```
Comment (0xKrishna) [inline cmd/geth/consolecmd.go:140]: Since mumbai and mainnet have same path, Lets use OR here and this condition with mumbai flag only

File: core/genesis.go [hunk 2]
```diff
@@ -417,6 +419,23 @@ func DefaultMumbaiGenesisBlock() *Genesis {
...
+func DefaultBorMainnetGenesisBlock() *Genesis {
+	return &Genesis{
+		Config:     params.BorMainnetChainConfig,
+		Nonce:      0x0,
+		Timestamp:  0x5ED20F84,
+		GasLimit:   0x989680,
+		Difficulty: big.NewInt(1),
+		Mixhash:    common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000000"),
+		Coinbase:   common.HexToAddress("0x0000000000000000000000000000000000000000"),
+		Alloc:      readPrealloc("allocs/bormainnet.json"),
```
Comment (0xKrishna) [inline core/genesis.go:432]: Please rename this to bor_mainnet.json

File: cmd/geth/consolecmd.go (hunk 1 above)
Comment (Uttam-Singhh) [reply cmd/geth/consolecmd.go]: Done!

File: cmd/geth/consolecmd.go [hunk 3]
```diff
@@ -134,12 +134,12 @@ func remoteConsole(ctx *cli.Context) error {
 				path = filepath.Join(path, "rinkeby")
 			} else if ctx.GlobalBool(utils.GoerliFlag.Name) {
 				path = filepath.Join(path, "goerli")
-			} else if ctx.GlobalBool(utils.MumbaiFlag.Name) {
+			} else if ctx.GlobalBool(utils.MumbaiFlag.Name) || ctx.GlobalBool(utils.BorMainnetFlag.Name) {
 				homeDir, _ := os.UserHomeDir()
 				path = filepath.Join(homeDir, "/.bor/data")
 			}
+			endpoint = fmt.Sprintf("%s/geth.ipc", path)
```
Comment (0xKrishna) [inline cmd/geth/consolecmd.go:141]: Why this change ?

Comment (Uttam-Singhh) [reply cmd/geth/consolecmd.go]: Resolved!
